src/services/codefixes/importFixes.ts

```diff
@@ -0,0 +1,393 @@
+                        if (namedBindings) {
+                            if (namedBindings.kind === SyntaxKind.NamespaceImport) {
+                                return getCodeActionForNamespaceImport(namedBindings.name.getText());
+                            }
+                            /**
+                             * If the existing import declaration already has a named import list, just
+                             * insert the identifier into that list.
+                             */
+                            const textChange = getTextChangeForImportList(namedBindings);
+                            return createCodeAction(
+                                Diagnostics.Add_0_to_existing_import_declaration_from_1,
+                                [name, declaration.moduleSpecifier.getText()],
+                                textChange.newText,
+                                textChange.span,
+                                sourceFile.fileName
+                            );
+                        }
+
+                        // insert a new import statement in a new lines
+                        return getCodeActionForNewImport(declaration.moduleSpecifier.getText(), declaration.getEnd());
+                    }
+                    return getCodeActionForNamespaceImport(declaration.name.getText());
+
+                    function getTextChangeForImportList(importList: NamedImports): TextChange {
+                        if (importList.elements.length === 0) {
+                            const start = importList.getStart();
+                            return {
+                                newText: `{ ${name} }`,
+                                span: { start, length: importList.getEnd() - start }
+                            };
+                        }
+
+                        // Insert after the last element
+                        const insertPoint = importList.elements[importList.elements.length - 1].getEnd();
+
+                        // If the import list has one import per line, preserve that. Otherwise, insert on same line as last element
+                        let oneImportPerLine: boolean;
+                        if (importList.elements.length === 1) {
+                            /**
+                             * If there is only one symbol being imported, still check to see if it's set up for multi-line imports like this:
+                             *     import {
+                             *         foo
+                             *     } from "./module";
+                             */
+                            const startLine = getLineOfLocalPosition(sourceFile, importList.getStart());
+                            const endLine = getLineOfLocalPosition(sourceFile, importList.getEnd());
+                            oneImportPerLine = endLine - startLine >= 2;
+                        }
+                        else {

```

Comment:
(and actually, you already return on line 115 if length was 0, so you don't even need an "if" check).